### PR TITLE
[API-692] Clean up documentation for athletes endpoints

### PR DESCRIPTION
**Pull request by hariniriyer**
Added the sample responses for the GET and PUT endpoints for the athlete.

### CODE REVIEW

**Comment by hariniriyer** [comment]
@zackisaacs PTAL and let me know if I missed to stub out something.

File: swagger/swagger.json.mustache [hunk 1]
```diff
@@ -99,6 +99,55 @@
             "description": "Profile information for the logged-in athlete",
             "schema": {
               "$ref": "{{reference_prefix}}/athlete.json#/DetailedAthlete"
+            },
+            "examples": {
+                "application/json": {
+                     "id": 12345,
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: This is a real athlete. Let's put a bigger ID here that will never correspond to a real athlete, like `12345678987654321`. Any number several times bigger than the population of the world should do 😉

File: swagger/swagger.json.mustache [hunk 2]
```diff
@@ -99,6 +99,55 @@
...
+                     "lastname": "Doe",
+                     "city": "San Francisco",
+                     "state": "CA",
+                     "country": "US",
+                     "sex": "F",
+                     "premium": true,
+                     "created_at": "2017-11-14T02:30:05Z",
+                     "updated_at": "2018-02-06T19:32:20Z",
+                     "badge_type_id": 4,
+                     "profile_medium": "https://xxxxxx.cloudfront.net/pictures/athletes/12345/67890/2/medium.jpg",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: Let's stub out the IDs in the photos here and in the line below, e.g.
```
"profile_medium": "https://xxxxxx.cloudfront.net/pictures/athletes/123456789/123456789/2/medium.jpg"
```

File: swagger/swagger.json.mustache [hunk 3]
```diff
@@ -99,6 +99,55 @@
...
+                     "mutual_friend_count": 0,
+                     "athlete_type": 1,
+                     "date_preference": "%m/%d/%Y",
+                     "measurement_preference": "feet",
+                     "clubs": [],
+                     "ftp": null,
+                     "weight": 0,
+                     "bikes": [
+                     {
+                        "id": "b241",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: Let's keep the IDs to be strictly numbers, but just make sure they're very large.

File: swagger/swagger.json.mustache [hunk 4]
```diff
@@ -99,6 +99,55 @@
...
+                        "primary": true,
+                        "name": "EMC",
+                        "resource_state": 2,
+                        "distance": 0
+                     }
+                     ],
+                     "shoes": [
+                     {
+                        "id": "g2106",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: Let's make this a large number.

File: swagger/swagger.json.mustache [hunk 5]
```diff
@@ -133,6 +182,55 @@
             "description": "Profile information for the logged-in athlete",
             "schema": {
               "$ref": "{{reference_prefix}}/athlete.json#/DetailedAthlete"
+            },
+            "examples": {
+                "application/json": {
+                    "id": 12345,
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: `12345678987654321`

File: swagger/swagger.json.mustache [hunk 6]
```diff
@@ -133,6 +182,55 @@
...
+                    "city": "San Francisco",
+                    "state": "CA",
+                    "country": "US",
+                    "sex": "F",
+                    "premium": true,
+                    "created_at": "2017-11-14T02:30:05Z",
+                    "updated_at": "2018-02-06T19:32:20Z",
+                    "badge_type_id": 4,
+                    "profile_medium": "https://xxxxxx.cloudfront.net/pictures/athletes/12345/67890/2/medium.jpg",
+                    "profile": "https://xxxxx.cloudfront.net/pictures/athletes/12345/67890/2/large.jpg",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: Same thing here -- let's sanitize the IDs in these pictures.

File: swagger/swagger.json.mustache [hunk 7]
```diff
@@ -133,6 +182,55 @@
...
+                    "mutual_friend_count": 0,
+                    "athlete_type": 1,
+                    "date_preference": "%m/%d/%Y",
+                    "measurement_preference": "feet",
+                    "clubs": [],
+                    "ftp": null,
+                    "weight": 0,
+                    "bikes": [
+                    {
+                        "id": "b241",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: Let's make this a number.

File: swagger/swagger.json.mustache [hunk 8]
```diff
@@ -133,6 +182,55 @@
...
+                        "primary": true,
+                        "name": "EMC",
+                        "resource_state": 2,
+                        "distance": 0
+                    }
+                    ],
+                    "shoes": [
+                    {
+                       "id": "g2106",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: Let's make this a number.

File: swagger/swagger.json.mustache (hunk 4 above)
Comment (hariniriyer) [reply swagger/swagger.json.mustache]: This is how these are stored on our end. This is a stubbed out id.

File: swagger/swagger.json.mustache (hunk 3 above)
Comment (hariniriyer) [reply swagger/swagger.json.mustache]: This is how these are stored on our end. This is a stubbed out id.

**Comment by hariniriyer** [comment]
@zackisaacs PTAL

File: swagger/swagger.json.mustache [hunk 9]
```diff
@@ -91,16 +91,65 @@
     "/athlete": {
       "get": {
         "operationId": "getLoggedInAthlete",
-        "summary": "Get Logged-in Athlete",
-        "description": "Returns the currently logged-in athlete",
+        "summary": "Get Authenticated Athlete",
+        "description": "Returns the currently authenticated athlete",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: nit: I believe we decided on periods for descriptions 😉

File: swagger/swagger.json.mustache [hunk 10]
```diff
@@ -114,7 +163,7 @@
       "put": {
         "operationId": "updateLoggedInAthlete",
         "summary": "Update Athlete",
-        "description": "Update the currently logged-in athlete",
+        "description": "Update the currently authenticated athlete",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: Period.

File: swagger/swagger.json.mustache [hunk 11]
```diff
@@ -131,9 +180,58 @@
         ],
         "responses": {
           "200": {
-            "description": "Profile information for the logged-in athlete",
+            "description": "Profile information for the authenticated athlete",
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: Period.

**Comment by hariniriyer** [comment]
@MateoAO @zackisaacs I changed the name to Marianne V and I changed all the descriptions in the file to have a period at the end (the ones that did not). PTAL if you want. I will merge it tomorrow.

**Comment by MateoAO** [comment]
👍 don't forget to squash

**Pull request merged**